edx_proctoring/__init__.py

```diff
@@ -5,6 +5,6 @@
 from __future__ import absolute_import
 
 # Be sure to update the version number in edx_proctoring/package.json
-__version__ = '2.1.9'
+__version__ = '2.2.0'
```

Comment:
nit: point releases can go higher than 9 - unless you really do want a new minor version for this release.

Reply:
I agree, but I think it's ok to do a minor version here to keep this clean.

Reply:
👍